### DISCUSSION TITLE
Why this javascript  custom search not work?

### DISCUSSION THREAD

**Discussion by OriLiMu** [gdh1995/vimium-c#1159] [Q&A] [answered]
I the custom search engine. I have the code : 
```
t: javascript:\ alert('hello'); JavaScript
m: javascript:\ (async () => { try { await navigator.clipboard.writeText(`[${document.title}](${location.href})`); console.log('Markdown link copied to clipboard.'); } catch (err) { console.error('Failed to copy markdown link to clipboard:', err); } })(); JavaScript
```
This code works fine in dev console. 
Use m to copy the current url in markdown mode. But nothing happens when I trigger it. 
I can trigger simple code. Like alert('hello')

**Comment by gdh1995** [comment, marked as answer]
the spaces must be encoded (`%20`)or escaped (like the one in `javascript:\ alert`).

or save it as a bookmark and then `map xxx openBookmark title="the-exact-title-of-a-target-bookmark"`
